pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-data-export</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
I think this has to be 1.0.0-SNAPSHOT. while releasing we set it to 1.0.0

Reply:
done